### PR TITLE
added the ability to automatically detect the extension of the currents.config file and to look for the right file

### PR DESCRIPTION
**Pull request by I-vac**

### CODE REVIEW

**Comment by I-vac** [comment]
Had an issue with the currents.config file not loading, because in my project I use type:modules and the file had to be .cjs, but it was hardcoded to look only for .js . Saw some other people having the same issue as me and decided to try and fix it myself.

**Comment by agoldis** [comment]
Thanks a lot @I-vac will take a look shortly

**Review by agoldis — CHANGES_REQUESTED** [review]
Thanks for submitting the PR @I-vac and sorry for taking so long to review. Please see my inline comments.

File: packages/cypress-cloud/lib/config/config.ts [hunk 1]
```diff
@@ -95,5 +96,18 @@ export async function getMergedConfig(params: ValidatedCurrentsParameters) {
...
+  for (const extension of extensions) {
+    const filepath = path.join(projectRoot ?? process.cwd(), `${filename}.${extension}`);
+    if (fs.existsSync(filepath)) {
+      filepaths.push(filepath);
+    } else {
+      console.warn(`${filepath} does not exist.`);
+    }
+  }
+
+  return filepaths;
```
Comment (agoldis) [inline packages/cypress-cloud/lib/config/config.ts:112]: Please take a look at [const fsConfig = require(resolvedPath);](https://github.com/currents-dev/cypress-cloud/blob/0599cba23af142b458da8cb690fac90a9b6bff82/packages/cypress-cloud/lib/config/config.ts#L46-L47)

- the return value should be a valid argument to `path.resolve` - an array of a path elements
- the resolved file can be `require`'d as a commonJS module
- 
A good validation can be a test that makes sure we can actually require different extenstions.

**Pull request closed without merging**